index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+        <!-- <section class="box header">
+        <div class="top_section">
+            <p class="time">9:41</p>
+            <ul>
+                <li><img src="Media/Cellular Connection.png" alt="Network Cell"></li>
+                <li><img src="Media/Wifi.png" alt="wifi"></li>
+                <li><img src="Media/Battery.png" alt="Battery"></li>
+            </ul> 
+        </div>
+        <nav class="header">
+            <a href="#" class="logo">
+                <h2>My Logo</h2>
+            </a>
+            <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_item2"></a> -->
+            <!-- <nav class="menu_icon">
+                <a href="#" class="Logo"><h2>My Logo</h2></a>
+                    <a href="#" class="menu_item2"><img src="Media/Union.svg" alt="Menu"></a>
+            </nav> -->
+            <header class="header">
+               <a href="#" class="logo">
+                <h2>My Logo</h2>
+               </a> 
+               <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_iteme2">></a>
```

Comment:
- [ ] As mentioned by the previous reviewer #1 :point_left: I think it would be better if you wrap the hamburger icon in a `nav` tag, doing this will improve the accessibility of your webpage since users using web accessibility tools like screen readers will be able to identify where the navigation on the webpage is. check below :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
         <nav>
               <a href="#" class="logo">
                <h2>My Logo</h2>
               </a> 
               <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_iteme2">></a>
         </nav>
```